### ISSUE TITLE
No support of reportportal plugin with pytest-3.1.2 while version pytest-3.0.7 is in requirements

### ISSUE THREAD

**Issue by itepliuk** [reportportal/agent-python-pytest#138]
An issue has occured while trying to integrate plugin into existing project with pytest==3.1.2 
My project has platform darwin -- Python 3.7.4, pytest-3.1.2, py-1.8.0, pluggy-0.4.0

> - What are minimal requirements?

**Expected:** 
Up to date requirements info. I expected to be able to use `--reportportal`.

**Actual:** 
After successful installation of plugin: 
`pip install pytest-reportportal`
running the test raises error:
`py.test ./tests` ->
```
File "/Users/#####/code/penv/lib/python3.7/site-packages/_pytest/assertion/rewrite.py", line 215, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File "/Users/#####/code/penv/lib/python3.7/site-packages/pytest_reportportal/service.py", line 10, in <module>
    from _pytest.nodes import File, Item
ModuleNotFoundError: No module named '_pytest.nodes'
```

Minimal project specification now was found here: https://github.com/reportportal/agent-python-pytest/blob/master/setup.py
```
requirements = [
    'reportportal-client>=3.2.3',
    'pytest>=3.0.7',
    'six>=1.10.0',
    'dill>=0.2.7.1',
]
```
Works fine with pytest-4.0.0, but that doesn`t work for my project.

**Comment by iivanou** [comment]
@itepliuk Could you try to apply changes in #139 locally and check how are things going?

**Comment by itepliuk** [comment]
Hi, @iivanou, for 3.1.2 that fix just helped for one import from `_pytest.nodes`, but the next import error appeared: 
```
  File "/Users/#####/code/penv/lib/python3.7/site-packages/pytest_reportportal/service.py", line 17, in <module>
    from _pytest.warning_types import PytestWarning
ModuleNotFoundError: No module named '_pytest.warning_types'
```

**Comment by iivanou** [comment]
@itepliuk Can you apply fixes from #141 and try again?

**Comment by itepliuk** [comment]
> @itepliuk Can you apply fixes from #141 and try again?

Sure, I applied that fix, now there are no import errors for pytest-3.1.2. Thanks
